python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -428,7 +431,7 @@ def test_series_init_none():
 
 
 def test_dataframe_basic():
-    np.random.seed(0)
+    rng = np.random.default_rng(seed=0)
```

Comment:
Having set a global `rng`, why produce a local one here?

Reply:
Removed the global one, the global one was only for the parametrization parts though.